packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import styles from './Card.module.css';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+
+export function RaisedLabel({
+  trending,
+  listMode,
+  type,
+}: {
+  trending: number;
+  listMode?: boolean;
+  type: 'hot' | 'pinned';
+}): ReactElement {
+  const description = `${trending} devs read it last hour`;
+  return (
+    <div
+      className={classNames(
+        'absolute flex items-start',
+        listMode ? 'top-0 right-full mt-5' : 'left-0 bottom-full ml-5 h-5',
+      )}
+    >
+      <SimpleTooltip content={description}>
+        <div
+          className={classNames(
+            'flex items-center px-1',
+            styles.flag,
```

Comment:
Just a question can we solve this with Tailwind possibly? 
If not it's fine, but prefer to eventually get rid of these.

Reply:
After investigating we are using the css module as of the transform and transition properties we are using are not support by tailwind